recipes/procps/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "procps" %}
+{% set version = "3.3.12" %}
+{% set sha256 = "b1036c109f271c7c50325b11a748236f8e58aa1dbafefb30c995ff1d05b4a1a8" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://gitlab.com/procps-ng/procps/repository/v3.3.12/archive.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
```

Comment:
Can you switch to two-space indentation, for consistency with the rest of conda-forge?

Reply:
Argh. bioconda asked me to switch to 4 space indentation.. So I did.